aiobotocore/aiomoto/aws_fixtures.py

```diff
@@ -0,0 +1,92 @@
+"""
+AWS test fixtures
+"""
+import os
+
+import boto3
+import pytest
+
+from moto import mock_batch
+from moto import mock_ec2
+from moto import mock_ecs
+from moto import mock_iam
+from moto import mock_logs
+from moto import mock_s3
+
+from aiobotocore.aiomoto.utils import AWS_REGION
+from aiobotocore.aiomoto.utils import AWS_ACCESS_KEY_ID
+from aiobotocore.aiomoto.utils import AWS_SECRET_ACCESS_KEY
+
+AWS_HOST = "127.0.0.1"
+AWS_PORT = "5000"
+
+
+@pytest.fixture
+def aws_host():
+    return os.getenv("AWS_HOST", AWS_HOST)
+
+
+@pytest.fixture
+def aws_port():
+    return os.getenv("AWS_PORT", AWS_PORT)
```

Comment:
should use a random port or else you can't run tests in parallel

Reply:
Pushed a new commit to try to use a free port, although I'm not sure the patch will be used in moto.